cpp/change-notes/2020-09-29-range-analysis-rollup.md

```diff
@@ -0,0 +1,14 @@
+lgtm,codescanning
```

Comment:
Where do these tags come from?

Reply:
They're product names. The release notes are generated by running a script (in an internal repo), passing a pair of SHAs/branches and a product name. The script will concatenate all release notes tagged with that product name from files that have been added between the two SHAs.

I believe the dates in the file names are only a convention to prevent conflicts (yay -- no more merge conflicts in change notes :heart_eyes:).

Reply:
So when I'm writing a change note under the new system, how do I know which tags to apply?  Is it always these two (for C/C++ QL changes at the present time)?

Reply:
> (yay -- no more merge conflicts in change notes 😍).

Much appreciated!